components/signoffs/signoffController.js

```diff
@@ -26,6 +26,20 @@ const getSignoff = async (req, res, next) => {
   }
 };
 
+const getAllSignoffs = async (req, res, next) => {
+  try {
+    //authenticate api-key from header before continuing
+    let apiKey = req.header('X-API-KEY');
+    const keyExists = await authenticator.authenticateKey(apiKey);
```

Comment:
Could add an `authenticateRequest` function to `apiAuth` so you don't need to pull the header each time

Reply:
Good idea!